src/vos/vos_dtx.c

```diff
@@ -573,7 +574,7 @@ dtx_ilog_rec_release(struct umem_instance *umm, struct vos_container *cont,
 
 	ilog_close(loh);
 
-	if (rc != 0)
+	if (rc != 0 && rc != -DER_NONEXIST)
```

Comment:
Isn't it too late to check DER_NONEXIST here? I suppose error is already returned when above ilog_open() failed, right?

Reply:
Such log message is no matter, its caller `do_dtx_rec_release()` will print.

Reply:
It's better to add an error message in ilog_open() or after the ilog_open() call above.